kythe-browse.sh

```diff
@@ -34,18 +37,11 @@ rm -f -- ${KYTHE_OUT}/graphstore/* ${KYTHE_OUT}/tables/*
 mkdir -p ${KYTHE_OUT}/graphstore ${KYTHE_OUT}/tables
 bazel build -c opt //verilog/tools/kythe:all
 
-for i in "$@"; do
-  echo "$(basename $i)" > "$(dirname $i)/file_list.txt"
-  cat "$(dirname $i)/file_list.txt"
-  # Read JSON entries from standard in to a graphstore.
-  bazel-bin/verilog/tools/kythe/verible-verilog-kythe-extractor --file_list_path "$(dirname $i)/file_list.txt" --file_list_root "$(dirname $i)"  --print_kythe_facts json  > "${KYTHE_OUT}"/entries
-
-  # Write entry stream into a GraphStore
-  "${KYTHE_BINDIR}"/entrystream --read_format=json < "${KYTHE_OUT}"/entries \
-  | "${KYTHE_BINDIR}"/write_entries -graphstore "${KYTHE_OUT}"/graphstore
-
-  rm "$(dirname $i)/file_list.txt"
-done
+# Read JSON entries from standard in to a graphstore.
+bazel-bin/verilog/tools/kythe/verible-verilog-kythe-extractor --file_list_path "$1" --file_list_root "$2" --print_kythe_facts json --include_dir_paths "$3" > "${KYTHE_OUT}"/entries
```

Comment:
This path assumes that the script is to be run from the Verible project root.
Declare this assumption in `usage()` (along with any others).